src/react-components/layout/Footer.js

```diff
@@ -25,11 +25,11 @@ export function Footer({
           {!hidePoweredBy && (
             <FormattedMessage
               id="footer.powered-by"
-              defaultMessage="Powered by <a>Mozilla Hubs</a>"
+              defaultMessage="Powered by <a>Hubs</a>"
               values={{
                 // eslint-disable-next-line react/display-name
                 a: chunks => (
-                  <a className={styles.link} href="https://hubs.mozilla.com">
+                  <a className={styles.link} href="https://demo.hubsfoundation.org">
```

Comment:
`demo.hubsfoundation.org` will likely link to the official instance for the Hubs Foundation, but I think it probably makes more sense to link to the main website here instead, so I would just link to `hubsfoundation.org`

Reply:
I just made another commit based on you comments, do I have to create another pull request or it would be fine?

Reply:
Nope, this is good.  Thanks!